mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -72,7 +72,9 @@ object CompletionValue:
   case class Keyword(label: String, insertText: String) extends CompletionValue
 
   def fromCompiler(completion: Completion): List[CompletionValue] =
-    completion.symbols.map(Compiler(completion.label, _))
+    def undoBacktick(label: String): String =
+      label.stripPrefix("`").stripSuffix("`")
```

Comment:
I'm assuming these are due to my changes in https://github.com/lampepfl/dotty/pull/14594. Is this how we want to handle this though? In these cases aren't we stripping them here and then adding them back later? Would it be better to instead check when we are about to add them and if they exist already, then don't add them?

Reply:
@ckipp01 I agree that it isn't perfect but they anyway should be removed from completion label